src/fabric/src/fabric2_fdb.erl

```diff
@@ -367,31 +357,39 @@ get_info(#{} = Db) ->
     [CProp | MProps].
 
 
-get_config(#{} = Db) ->
+load_config(#{} = Db) ->
     #{
         tx := Tx,
         db_prefix := DbPrefix
-    } = ensure_current(Db),
+    } = Db,
 
     {Start, End} = erlfdb_tuple:range({?DB_CONFIG}, DbPrefix),
     Future = erlfdb:get_range(Tx, Start, End),
 
-    lists:map(fun({K, V}) ->
+    lists:foldl(fun({K, V}, DbAcc) ->
         {?DB_CONFIG, Key} = erlfdb_tuple:unpack(K, DbPrefix),
-        {Key, V}
-    end, erlfdb:wait(Future)).
+        case Key of
+            <<"uuid">> ->  DbAcc#{uuid => V};
```

Comment:
Should probably make these `:=` to assert we've not typo'd the keys.

Reply:
Good idea. I didn't set since uuid was missing. After chatting in Slack, will set uuid to <<>> and then switch to using :=